src/Data/Maybe/Categorical.agda

```diff
@@ -8,34 +8,43 @@
 
 module Data.Maybe.Categorical where
 
+open import Level
 open import Data.Maybe.Base
 open import Category.Functor
 open import Category.Applicative
 open import Category.Monad
 import Function.Identity.Categorical as Id
 open import Function
 
+private
+  variable
+    a b f m : Level
+    A : Set a
+    B : Set b
+    F : Set f → Set f
```

Comment:
So the current [policy](https://github.com/agda/agda-stdlib/blob/master/notes/style-guide.md#variables) for `variable`s is that they should only be declared over levels/sets and the "main" types that the file concerns itself with, which in this case is `Maybe`. So I don't think `F` and `M` qualify under this policy. Indeed they are only used once in the file at the moment, so I think far better to keep them local!

Otherwise happy to keep the levels and the sets!